test.py

```diff
@@ -297,9 +308,18 @@ def run_tests(
     options=[],
     interop_tests=False,
     workers=None,
+    only_pattern=None,
 ):
     if interop_tests:
-        legate_tests.extend(glob.glob("tests/interop/*.py"))
+        legate_tests.extend(glob.glob("tests/interop/*.py"))  # noqa: F823
```

Comment:
Why do you need to override F823? We generally don't want overrides like this, unless there's no way around it. (as far as I know we use this override only for very long function names.)